packages/client/src/components/TestToolBox.js

```diff
@@ -62,9 +32,11 @@ const TestToolBox = ({ address }) => {
     setUserFUSDBalance(fusdBalance);
     setUserFlowBalance(flowBalance);
   };
+
```

Comment:
useEffect defined without dependency array will cause the callback function to run on each rendering. The purpose of the empty array here is to only run it once when component mounts. Since we have the refresh button, I'd say let's keep the empty dependency array to avoid unnecessary data fetching on most rendering cycles.

Reply:
If I remove it there is a warning from the compiler: React Hook useEffect has a missing dependency: 'updateUserBalance'. Either include it or remove the dependency array. 
Should we ignore the warning?

Reply:
```suggestion
  useEffect(() => {
    updateUserBalance();
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```
we can disable eslint for that line, it will make the warning go away